packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss

```diff
@@ -5,7 +5,7 @@
   box-sizing: content-box;
   flex-shrink: 1;
   background: $ms-color-neutralLighter;
-  margin: 1px;
+  margin: 2px;
```

Comment:
If this should be consistent with the peoplepickeritems shouldn't the margin match at `1px 2px`?

Reply:
@joschect, TagItems have a 2px smaller height than peoplepickeritems, so 1px smaller margins are needed to fit them in the same 30px container. The horizontal margins between them both is consistent at 2px. Is this OK, or should the TagItems be made taller so the height and margins match on both?

Reply:
That's fine, I just wanted to make sure there was a reason!